src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -70,6 +70,8 @@
     private static final String ANDROID_TRIPLET = new Triplet(Constants.Profile.ANDROID).toString();
     private static final String ANDROID_MIN_SDK_VERSION = "21";
     private static final List<String> ANDROID_KEYSTORE_EXTENSIONS = List.of(".keystore", ".jks");
+    public static final String WL_WHOLE_ARCHIVE = "-Wl,--whole-archive";
+    public static final String WL_NO_WHOLE_ARCHIVE = "-Wl,--no-whole-archive";
```

Comment:
Any specific reason why these are public? I don't see their usage outside this class.

Reply:
Fixed